script/1_0_2current_en_script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -353,7 +353,7 @@ Senpai forces herself to act cheerful.
 This topic might have been fine for a long-time friend like Arihiko but for someone like Senpai, such a complex topic must've been bothersome at best.
 In fact, she seems to be fidgeting as if she is uncomfortable.
 <Page111>
-"Aah, I'm sorry, Senpai. But I've got to talk to Tohno about something in private so can you excuse us for a sec?"
+"Aah, I'm sorry, Senpai. But I've got to talk to Tohno about something in private, so could you please give us some space?"//「あー、ゴメン先輩。オレ、ちょっと遠野と内緒話があるからさ、席外してもらえない？」 "Can you excuse us" implies that they (Shiki & Arihiko) are the ones leaving right? If so, that's the contrary
```

Comment:
I think that the original leaves it ambiguous and sounds good, I think maybe revert.

Reply:
I'll revert then.